### ISSUE TITLE
Review Approach 

### ISSUE THREAD

**Issue by smellems** [sara-sabr/ITStrategy#836]
https://github.com/sara-sabr/ITStrategy/blob/master/_pages/en/web-services-access.md
is asking to unblock specific sites.

We now know that requesting Riot.im or Google Docs (probably many others), will result in most sites for IM, Document Editing and Web conference being unblocked for users. Email and File Storage remain mostly blocked.  Websockets are still blocked (except for Edge).

Maybe it would be simpler to ask for this to be the standards access for IITB employees.

We could also ask to unblock categories of sites instead of specific sites.

Related to #835 & #880

**Comment by remyb2canada** [comment]
Multiple milestones relate to each other:
1- [open access to online tools](https://github.com/sara-sabr/ITStrategy/milestone/8)
2- [desktop tools](https://github.com/sara-sabr/ITStrategy/milestone/37)
3- [POC - software onboarding](https://github.com/sara-sabr/ITStrategy/milestone/18)

**Comment by jaysonmc** [comment]
Meeting took place

Produced new issue: #934
